modules/sdk-core/src/bitgo/utils/postWithCodec.ts

```diff
@@ -0,0 +1,77 @@
+import * as t from 'io-ts';
+import { BitGoBase } from '../bitgoBase';
+import { BitGoRequest } from '../../api';
+import { SuperAgent, SuperAgentRequest } from 'superagent';
+import { isLeft } from 'fp-ts/Either';
+
+/**
+ * @param body
+ * @param encodedBody
+ * @returns a list of unknown properties that are present in the body but not the codec.
+ */
+function getUnknownProperties(body: Record<string, unknown>, encodedBody: Record<string, unknown>): string[] {
+  const unknownProperties: string[] = [];
+  if (body && encodedBody) {
+    const bodyKeys = Object.keys(body);
+    const encodedBodyKeys = Object.keys(encodedBody);
+    const unknownKeys = bodyKeys.filter((key) => !encodedBodyKeys.includes(key));
+    unknownProperties.push(...unknownKeys);
```

Comment:
```suggestion
    unknownProperties = unknownKeys.map((k) => body[k]);
```
This might make more sense?